src/movies.html

```diff
@@ -26,9 +28,190 @@
 
 <main class="main">
     <div class="container">
-
+        <div class="place-right">
+            <button class="basic-btn" id="filter-btn">Filters</button>
+        </div>
+        <div class="movies">
+            <img class="bubble right" src="./images/films/round.svg" alt="Logo"></a>
+            <img class="bubble left" src="./images/films/round.svg" alt="Logo"></a>
```

Comment:
maybe add these elements as pseudo-elements in css for div.movies
and create them with only css (not img)